### DISCUSSION TITLE
Crashing when playing some VOBs on Roku

### DISCUSSION THREAD

**Discussion by ricks03** [jellyfin/jellyfin#11110] [General]
Hi! I'm troubleshooting why some of my files will play on the Roku interface, and others will not. 

I am running Jellyfin 10.8.10 on CentOS, and it's generally set up and working fine. I can view videos on the web UI, and I can view videos on the Android client, and on the Roku client. However, for a few of my videos, they won't play on the Roku Jellyfin app. Instead, the Roku app crashes back to the main Roku screen.

The server has integrated video, and does not have hardware acceleration enabled (although I tested this problem with V4L but the same results). The MB is a asus m2npv-vm and running an AMD Phenom x4. 

I have a number of shows I have ripped to VOB, that are arranged in /Season 1 , /Season 2 folders etc. Some of the shows work on Roku, some do not.  All appear to play on the Jellyfin Android app  and browsing the server remotely via Chrome. 

For those that do not work, I select the show, and the Roku Jellyfin app crashes back to the main Roku UI. 

I don't have any problem playing AVI, mkv, mp4, or VOB, except this small subset of the VPB files.

**Comment by 1hitsong** [comment]
This isn't a good place to ask for troubleshooting help. Instead, I suggest posting to the Troubleshooting forum to ask for help.

https://forum.jellyfin.org/f-troubleshooting

**Comment by ricks03** [comment]
I'm not really looking for troubleshooting help, but reporting an issue. 

When an app completely crashes, something is broken.
